### PR TITLE
adds card management page feature

### PR DESCRIPTION
**Pull request by AlicanAkkus**
* card management page initialize feature has added.

### CODE REVIEW

File: src/main/java/com/iyzipay/model/CardManagementPageInitialize.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import org.apache.commons.lang3.builder.ToStringStyle;
+
+public class CardManagementPageInitialize extends IyzipayResource {
+
+    private String externalId;
+    private String token;
+    private String cardPageUrl;
+
+    public static CardManagementPageInitialize create(CreateCardManagementPageInitializeRequest request, Options options) {
+        return HttpClient.create().post(options.getBaseUrl() + "/api/v1/forms",
```
Comment (yyenigun) [inline src/main/java/com/iyzipay/model/CardManagementPageInitialize.java]: Base url will be set like this: "https://api.iyzipay.com/api/v1/forms" ? If yes "api" words duplicated and we can not understand which form it is. You should write card management word to url. It should be https://api.iyzipay.com/v1/card-management/forms or https://cardapi.iyzipay.com/v1/forms.

Comment (AlicanAkkus) [reply src/main/java/com/iyzipay/model/CardManagementPageInitialize.java]: hi @yyenigun , baseUrl will be set like as in live environment "[https://api.iyzipay.com/api/v1/card-management/forms](https://api.iyzipay.com/api/v1/card-management/forms)"

Comment (yyenigun) [reply src/main/java/com/iyzipay/model/CardManagementPageInitialize.java]: @AlicanAkkus url should be like this: "https://api.iyzipay.com/v1/card-management/forms" and you should also change it in the code: options.getBaseUrl() + "/v1/card-management/forms"

Comment (AlicanAkkus) [reply src/main/java/com/iyzipay/model/CardManagementPageInitialize.java]: @yyenigun yo're right. I've changed it in the code.

File: src/main/java/com/iyzipay/model/CardManagementPageInitialize.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package com.iyzipay.model;
+
+import com.iyzipay.HttpClient;
+import com.iyzipay.IyzipayResource;
+import com.iyzipay.Options;
+import com.iyzipay.request.CreateCardManagementPageInitializeRequest;
+
+public class CardManagementPageInitialize extends IyzipayResource {
```
Comment (sotuzun) [inline src/main/java/com/iyzipay/model/CardManagementPageInitialize.java]: What is class name in the server? Maybe CardManagementFormInitialize could be the best.

Comment (AlicanAkkus) [reply src/main/java/com/iyzipay/model/CardManagementPageInitialize.java]: url has changed to pages, we will use this name for class.

**Pull request merged**